padle/padle.py

```diff
@@ -0,0 +1,537 @@
+                return True
+
+        await self.bot.wait_until_ready()
+        # idk what x is but it wants it to be there
+        async for x in conditional_iterator(is_day_change, poll_interval=10):
+            try:
+                # to be used at a later date...
+                async with self.config.save_daily_scores() as save_daily:
+                    save_daily.append([await self.config.padle_today(), await self.config.all_scores()])
+                with open("./pad-cogs/padle/monsters.txt", "r") as f:
+                    monsters = f.readline().split(",")
+                    await self.config.padle_today.set(random.choice(monsters))
+                num = await self.config.num_days()
+                await self.config.num_days.set(num + 1)
+                all_users = await self.config.all_users()
+                await self.config.all_scores.set([])
+                for userid in all_users:
+                    user = await self.bot.fetch_user(userid)
+                    await self.config.user(user).guesses.set([])
+                    # need to send message if a user is mid-game
+                    if(await self.config.user(user).start() and not await self.config.user(user).done()):
+                        await user.send("The PADle expired; a new one is available.")
+                    await self.config.user(user).start.set(False)
+                    await self.config.user(user).done.set(False)
+                    await self.config.user(user).score.set([])
+                    await self.config.user(user).edit_id.set("")
+                    await self.config.user(user).channel_id.set("")
+                    await self.config.user(user).embed_text.set([])
+                subbed_users = await self.config.subs()
+                for userid in subbed_users:
+                    user = await self.bot.fetch_user(userid)
+                    await user.send("PADle #{} is now available!".format(await self.config.num_days()))
+            except asyncio.CancelledError:
+                raise
+            except Exception:
+                logger.exception("Error in loop:")
+
+    async def get_name_line(self, ctx, monster, guess_monster):
+        line = []
+        attr1 = guess_monster.attr1.name.lower()
+        attr2 = guess_monster.attr2.name.lower()
+        attr_feedback = []
+        attr_feedback.append(":white_check_mark:" if attr1 == monster.attr1.name.lower() else ":x:")
+        attr_feedback.append(":white_check_mark:" if attr2 == monster.attr2.name.lower() else ":x:")
+        if(attr1 == monster.attr2.name.lower() and
+           attr2 != monster.attr2.name.lower() and attr1 != monster.attr1.name.lower()):
+            attr_feedback[0] = ":yellow_square:"
+        if(attr2 == monster.attr1.name.lower() and
+           attr2 != monster.attr2.name.lower() and attr1 != monster.attr1.name.lower()):
+            attr_feedback[1] = ":yellow_square:"
+        line.append(get_emoji("orb_{}".format(attr1)))
```

Comment:
You can get an dual_color orb emoji via `tsutils.tsubaki.custom_emoji.get_attribute_emoji_by_enum`

Reply:
![image](https://user-images.githubusercontent.com/68720253/171564618-5211c71e-862d-41f9-a2dc-3c7f5bd618b2.png)
I think I need the two orbs separate orbs to make it look nice with the ❌ and ✅

Reply:
You can use get_attribute_emoji_by_enum with just one attribute to get a full orb of that color if you'd like.  It makes it easier if we ever decide to change the emoji names, because it means we only have to change it in one place

Reply:
You can use get_attribute_emoji_by_enum with just one attribute to get a full orb of that color if you'd like. It makes it easier if we ever decide to change the emoji names, because it means we only have to change it in one place